### ISSUE TITLE
Telegram Flatpak - Proxy doesn't work

### ISSUE THREAD

**Issue by johntgd** [flathub/org.telegram.desktop#226]
Hello Community,

I just installed Telegram v2.5.1 from flatpak (https://flathub.org/apps/details/org.telegram.desktop) on my Xubuntu 20.04 system and **the proxy configuration seems to be either broken or unavailable within flatpak**.

Connection without any proxy works fine.

Just to clarify - on the very same system I'm running Telegram from the official Ubuntu repository (also v2.5.1) and the proxy with the exact same configuration (socks5 127.0.0.1 port 22) works just fine.

Any thoughts on what to look into?

Thank you in advance.

Btw. I'm running two installations as I want to have two isolated accounts or is this considered wrong approach/not the best practice?

**Comment by ilya-fedin** [comment]
> on the very same system I'm running Telegram from the official Ubuntu repository (also v2.5.1)

Looks like you got something wrong here. Official Ubuntu repositories for 20.04 contain version 2.1.7
https://packages.ubuntu.com/focal-updates/telegram-desktop

> Any thoughts on what to look into?

I have no idea, unfortunately... Maybe you should ask about network restrictions on the #flatpak IRC channel?

**Comment by gasinvein** [comment]
There are no network restriction with `--share=network`.
@johntgd How do you enanle proxy? In Telegram UI or in system settings?

**Comment by johntgd** [comment]
> Looks like you got something wrong here. Official Ubuntu repositories for 20.04 contain version 2.1.7
> https://packages.ubuntu.com/focal-updates/telegram-desktop

Thank you - you actually made me notice that I installed the Telegram from snap as default without even realizing that (v2.1.5) - as this is default in the gnome-software installer.
Installing secondary client via **apt install telegram-desktop** (which was the v2.1.7) actually solved the issue for me (so I now have two independent clients on 1 machine).

> I have no idea, unfortunately... Maybe you should ask about network restrictions on the #flatpak IRC channel?

I actually overlooked it and turned out that the ports were not 22 but something else - obviously my bad - so sorry to bother you with this.

Thank you for all your help.
